DefaultTabBar.js

```diff
@@ -45,7 +45,7 @@ const DefaultTabBar = React.createClass({
       accessibilityTraits='button'
       onPress={() => this.props.goToPage(page)}
     >
-      {this.renderTabName(name, page, isTabActive)}
+      {this.props.renderTabName(name, page, isTabActive)}
```

Comment:
Yes, I already started to do this. What about context for ScrollableTabBar?
Should we pass him to custom function somehow? I think, users should have
ability to call method measureTab or something else.

пт, 26 авг. 2016 г., 17:55 Evgeniy Sokovikov <notifications@github.com>:

> In DefaultTabBar.js
> <https://github.com/skv-headless/react-native-scrollable-tab-view/pull/402#discussion_r76432378>
> :
>
> > @@ -45,7 +45,7 @@ const DefaultTabBar = React.createClass({
> >        accessibilityTraits='button'
> >        onPress={() => this.props.goToPage(page)}
> >      >
> > -      {this.renderTabName(name, page, isTabActive)}
> > +      {this.props.renderTabName(name, page, isTabActive)}
>
> Pity. Then this.props.renderTab ? this.props.renderTab(name, page,
> isTabActive) : this.renderTab(name, page, isTabActive)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/skv-headless/react-native-scrollable-tab-view/pull/402/files/12a6026848a4edb731e0dc211d923a040757f368#r76432378>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAVoMPmUXH0ekjP94nEZ45jGpeNU9ztiks5qjv5TgaJpZM4JuAxf>
> .
>
-- 

-------------------------------------

С уважением,
Александр Клименков